src/App.js

```diff
@@ -51,7 +74,11 @@ function App() {
           setCurrentPage={setCurrentPage}
         />
         {recipe.length !== 0 && (
-          <DisplayRecipe recipe={recipe} onBookmark={handleBookmark} />
+          <DisplayRecipe
+            recipe={recipe}
+            onBookmark={handleBookmark}
+            bookmarks={bookmarks}
```

Comment:
At this point, I think you can already figure out whether the recipe is in the bookmark or not, so you don't have to send the whole `bookmarks` down the component hierarchy. A boolean check like `isRecipeBookmarked` would be sufficient instead 🤔